requirements.txt

```diff
@@ -1,9 +1,7 @@
 Sphinx
-myst-parser
+# myst-parser fixed the docutils warnings. Pinning to current SHA until
+# a new release greater than 2.0.0 is made.
+git+https://github.com/executablebooks/MyST-Parser.git@978e845543b5bcb7af0ff89cac9f798cb8c16ab3
```

Comment:
Maybe we can ping them to make a new release so that we don't have to pin the SHA?

Reply:
Left an issue and pinged Chris: https://github.com/executablebooks/MyST-Parser/issues/870

I'm going to merge the pin for now and we can remove after the release.